schemas/new.schema.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
And here is the JSON Schema, intended for a human (me) to work with, instead of the generated JSON Schema. Mostly, things are ordered in a more readable way.

Reply:
I like it! By now I can almost read JSON Schema, and I find this quite readable.